pandas/tests/tseries/test_timezones.py

```diff
@@ -882,6 +882,17 @@ def test_frame_reset_index(self):
         rs = roundtripped.index.tz
         assert xp == rs
 
+    def test_frame_to_dict_tz(self):
+        data = [(datetime(2017, 11, 18, 21, 53, 0, 219225, tzinfo=pytz.utc),),
```

Comment:
add the issue number as a comment.

this test goes in ``pandas/tests/frames/test_convert_to.py``